lib/python/vdm/server/Configuration.py

```diff
@@ -0,0 +1,606 @@
+import json
+import traceback
+from xml.etree.ElementTree import Element, SubElement, tostring, XML
+import HTTPListener
+import DeploymentConfig
+
+
+def convert_xml_to_json(config_path):
+    """
+    Method to get the json content from xml file
+    :param config_path (string): path of xml file
+    """
+    with open(config_path) as config_file:
+        xml = config_file.read()
+    config_content = XML(xml)
+    xml_final = json.loads(json.dumps(etree_to_dict(config_content)))
+    if type(xml_final['voltdeploy']['members']['member']) is dict:
+        member_json = get_field_from_xml(xml_final['voltdeploy']['members']['member'], 'dict')
+    else:
+        member_json = get_field_from_xml(xml_final['voltdeploy']['members']['member'], 'list')
+
+    if type(xml_final['voltdeploy']['databases']['database']) is dict:
+        db_json = get_field_from_xml(xml_final['voltdeploy']['databases']['database'],
+                                     'dict', 'database')
+    else:
+        db_json = get_field_from_xml(xml_final['voltdeploy']['databases']['database'],
+                                     'list', 'database')
+
+    if type(xml_final['voltdeploy']['deployments']['deployment']) is dict:
+        deployment_json = get_deployment_from_xml(xml_final['voltdeploy']['deployments']
+                                                  ['deployment'], 'dict')
+    else:
+        deployment_json = get_deployment_from_xml(xml_final['voltdeploy']['deployments']
+                                                  ['deployment'], 'list')
+    if type(xml_final['voltdeploy']['deployments']['deployment']) is dict:
+        user_json = get_users_from_xml(xml_final['voltdeploy']['deployments']['deployment'],
+                                       'dict')
+    else:
+        user_json = get_users_from_xml(xml_final['voltdeploy']['deployments']['deployment'],
+                                       'list')
+
+    HTTPListener.Global.DATABASES = db_json
+
+    HTTPListener.Global.SERVERS = member_json
+
+    HTTPListener.Global.DEPLOYMENT = deployment_json
+
+    HTTPListener.Global.DEPLOYMENT_USERS = user_json
+
+
+def get_deployment_from_xml(deployment_xml, is_list):
+    """
+    Function to get deployment json from the xml content
+    :param deployment_xml: raw deployment object
+    :param is_list: flag to know if it is list or dict
+    :return: deployment object in required format
+    """
+    deployments = []
+    if is_list is 'list':
+        for deployment in deployment_xml:
+            deployments.append(get_deployment(deployment))
+    else:
+        deployments.append(get_deployment(deployment_xml))
+    return deployments
+
+
+def get_deployment(deployment, is_upload=False):
+    """
+    Gets the required deployment object
+    :param deployment: raw deployment object
+    :param is_upload: flag to know if it is list or dict
+    :return: deployment object in required format
+    """
+    new_deployment = {}
+    for field in deployment:
```

Comment:
This method should be broken up into multiple ones. For example:
set_export_field(from_depl, to_depl)